google-cloud-firestore/src/test/java/com/google/cloud/firestore/MapperTest.java

```diff
@@ -1213,9 +1222,7 @@ public void primitiveDeserializeLong() {
 
   @Test
   public void primitiveDeserializeWrongTypeMap() {
-    assertExceptionContains(
-        "Failed to convert value of type java.util.LinkedHashMap to String "
-            + "(found in field 'value')",
+    assertExceptionThrows(
```

Comment:
This is overengineered. You can move the try-catch block into this method and not use a runnable.